app/views/unattended/report_templates/ansible_-_ansible_inventory.erb

```diff
@@ -158,8 +160,8 @@ require:
       inventory_data.update('location': host.location) if input('Location') == 'yes'
       hostgroup = host.hostgroup.title if host.hostgroup
       inventory_data.update('host_group': hostgroup) if input('Host Group') == 'yes'
-      host_collections = host.host_collections.map { |c| c.name } if host.host_collections
-      inventory_data.update('host_collections': host_collections) if input('Host Collections') == 'yes'
+      host_collections = host.host_collections.map { |c| c.name } if include_host_collections && host.host_collections
```

Comment:
I'd prefer to wrap these two lines in a single condition
```
if include_host_collections
  host_collections = host.host_collections.map { |c| c.name } if host.host_collections
  inventory_data.update('host_collections': host_collections)
end
```

also I think `host.host_colelctions` will be always truthy right? If there's none, it's an emtpy array, so I'm not sure what this condition is required for, but I know it's not added by this PR

Reply:
I think only managed hosts have that attribute, so maybe that's to guard against calling it on non-managed?

Reply:
I left the `if host.host_collections` as an exercise to whoever will edit this template after me.